fdbclient/NativeAPI.actor.cpp

```diff
@@ -3443,8 +3446,9 @@ ACTOR Future<Key> getKey(Reference<TransactionState> trState,
 	}
 }
 
-ACTOR Future<Version> waitForCommittedVersion(Database cx, Version version, SpanID spanContext) {
-	state Span span("NAPI:waitForCommittedVersion"_loc, { spanContext });
+ACTOR Future<Version> waitForCommittedVersion(Database cx, Version version, SpanContext spanContext) {
+	// TODO - ljoswiak, Parent or link? Below is linked constructor with empty parent.
```

Comment:
> Below is linked constructor with empty parent

What do you mean by this?

Reply:
Seems like it should be Parent? Not sure why I thought a link there at the time.